src/test/java/org/json/simple/Test.java

```diff
@@ -302,7 +302,7 @@ public void testEncode() throws Exception{
 		array1.add("abc\u0010a/");
 		array1.add(new Integer(123));
```

Comment:
You should use Integer.valueOf() instead of calling inefficient constructor, like you did below with the Boolean type.